src/app/space/add-space-overlay/add-space-overlay.module.ts

```diff
@@ -0,0 +1,39 @@
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { FormsModule } from '@angular/forms';
+
+import { Fabric8WitModule, UniqueSpaceNameValidatorDirective, ValidSpaceNameValidatorDirective } from 'ngx-fabric8-wit';
+
+import { CodebasesService } from '../create/codebases/services/codebases.service';
+import { TrustHtmlPipe, TrustStylePipe } from '../wizard/pipes/safe-html.pipe';
+import { SelectedItemsPipe } from '../wizard/pipes/selected-items.pipe';
+import { VisibleItemsPipe } from '../wizard/pipes/visible-items.pipe';
+import { AddSpaceOverlayComponent } from './add-space-overlay.component';
+
+@NgModule({
+  imports: [
+    CommonModule,
+    FormsModule,
+    Fabric8WitModule
+  ],
+  declarations: [
+    AddSpaceOverlayComponent,
+    UniqueSpaceNameValidatorDirective,
+    ValidSpaceNameValidatorDirective,
+    SelectedItemsPipe,
+    VisibleItemsPipe,
+    TrustHtmlPipe,
+    TrustStylePipe
+  ],
+  exports: [
+    AddSpaceOverlayComponent,
+    UniqueSpaceNameValidatorDirective
```

Comment:
Spent some time figuring out the build. Please remove the declarations and exports below and restore space-wizard.module.ts -- the UI will then build/run properly.

- UniqueSpaceNameValidatorDirective
- ValidSpaceNameValidatorDirective
- SelectedItemsPipe
- VisibleItemsPipe
- TrustHtmlPipe
- TrustStylePipe

Reply:
fixed